modules/OCPP/OCPP.cpp

```diff
@@ -102,16 +102,19 @@ void OCPP::set_external_limits(const std::map<int32_t, ocpp::v16::EnhancedChargi
         limits.schedule_import.emplace(schedule_import);
 
         if (connector_id == 0) {
-            if (!this->r_connector_zero_sink.empty()) {
+            if (!this->r_evse_manager_energy_sink.empty()) {
                 EVLOG_debug << "OCPP sets the following external limits for connector 0: \n" << limits;
-                this->r_connector_zero_sink.at(0)->call_set_external_limits(limits);
+                this->r_evse_manager_energy_sink.at(0)->call_set_external_limits(limits);
```

Comment:
`at(0) = first()`

Reply:
`front()` ;) but this access was removed